lnwallet/btcwallet/btcwallet.go

```diff
@@ -995,12 +992,55 @@ func (b *BtcWallet) SendOutputs(outputs []*wire.TxOut,
 		return nil, lnwallet.ErrInvalidMinconf
 	}
 
+	if selectedUtxos != nil {
+		return b.wallet.SendOutputsWithInput(
+			outputs, nil, defaultAccount, minConfs,
+			feeSatPerKB, b.cfg.CoinSelectionStrategy, label,
+			selectedUtxos,
+		)
+	}
+
 	return b.wallet.SendOutputs(
 		outputs, nil, defaultAccount, minConfs, feeSatPerKB,
 		strategy, label,
 	)
 }
 
+// SendOutputsWithInput funds, signs, and broadcasts a Bitcoin transaction using
+// selected utxos as input paying out to the specified outputs. In the case the
+// wallet has insufficient funds, or the outputs are non-standard, a non-nil
+// error will be returned.
+//
+// NOTE: This method requires the global coin selection lock to be held.
+//
+// This is a part of the WalletController interface.
+func (b *BtcWallet) SendOutputsWithInput(
+	outputs []*wire.TxOut, feeRate chainfee.SatPerKWeight, minConfs int32,
+	label string, strategy base.CoinSelectionStrategy,
+	selectedUtxos []wire.OutPoint) (*wire.MsgTx, error) {
+
+	return b.sendOutputs(
+		outputs, feeRate, minConfs, label, strategy, selectedUtxos,
+	)
+}
+
+// SendOutputs funds, signs, and broadcasts a Bitcoin transaction paying
+// out to the specified outputs. In the case the wallet has insufficient
+// funds, or the outputs are non-standard, an error will be returned.
+//
+// NOTE: This method requires the global coin selection lock to be held.
+//
+// This is a part of the WalletController interface.
+func (b *BtcWallet) SendOutputs(
```

Comment:
Looking at these three methods, I think we can simplify this by adding functional options to the `SendOutputs` method, similar to `...base.TxCreateOption` we have in the wallet. That way we don't need to add another whole new method to the interface just for a single additional parameter.